### DISCUSSION TITLE
can  we train Spacy to summarize text ?

### DISCUSSION THREAD

**Discussion by chamodSamajith** [explosion/spaCy#9262] [Help: Model Advice] [answered]
spacy can be trained for NER, as like that can we train spacy to summarize large paragraphs without directly using its functions? 
Training will help to improve the accuracy of the summarization

**Comment by polm** [comment, marked as answer]
So, there is not summarization component for spaCy. Summarization is also close enough to NLG that I think it's out of scope.
